db/models.py

```diff
@@ -14,3 +14,38 @@ class Actor(models.Model):
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+
+    actors = models.ManyToManyField(Actor, related_name="Movie")
+    genres = models.ManyToManyField(Genre, related_name="Movie")
+
+    def __str__(self) -> str:
+        return f"{self.title}"
+
+
+class CinemaHall(models.Model):
+    name = models.CharField(max_length=255)
+    rows = models.IntegerField()
+    seats_in_row = models.IntegerField()
+
+    @property
+    def capacity(self) -> int:
+        capacity = self.rows * self.seats_in_row
+        return capacity
+
+    def __str__(self) -> str:
+        return f"{self.name}"
```

Comment:
`__str__` method must be above `capacity` method

Reply:
Because magical methods always go first?